include/deal.II/lac/parpack_solver.h

```diff
@@ -913,7 +913,7 @@ void PArpackSolver<VectorType>::solve
           AssertThrow (false, PArpackExcInfoPdneupd(info));
         }
 
-      for (size_type i=0; i<nev; ++i)
+      for (size_type i=0; i<(size_type)nev; ++i)
```

Comment:
`nev` equals `n_eigenvalues`, the latter of which is an `unsigned int`. Why not make `i` an `unsigned int` as well, and avoid the warning that way?